### DISCUSSION TITLE
TV RiK [SK and CZ]

### DISCUSSION THREAD

**Discussion by sleduj-tv** [iptv-org/community#369] [Show and tell]
### Channel ID

TVRiK.sk

### Stream URL

http://91.237.170.202:11000/play/20211

### Notes

The stream is in quality 1920x1080.

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by AntiPontifex** [comment]
Thank you for the stream: it will be included in the next update.

**Comment by sleduj-tv** [comment]
when?

**Comment by AntiPontifex** [comment]
Whenever my pull request is approved: see appropriate section.

**Comment by sleduj-tv** [comment]
ok

**Comment by freearhey** [comment]
http://91.237.170.202:11000/play/20211 (offline)
<img width="278" alt="image" src="https://user-images.githubusercontent.com/7253922/232248268-87e1589e-a3b2-4790-8dc7-832f7efb528a.png">
